### ISSUE TITLE
dasolo.online 

### ISSUE THREAD

**Issue by torinoit** [reek/anti-adblock-killer#2888]
requires to disable adblock :(

**Comment by uBlock-user** [comment]
https://github.com/reek/anti-adblock-killer/issues/2802

**Comment by jspenguin2017** [comment]
No DNS record... 
![image](https://cloud.githubusercontent.com/assets/7283682/23333381/ce1c1f16-fb47-11e6-9e1d-206bf92c1294.png)

**Comment by uBlock-user** [comment]
That's not possible! 

https://who.is/whois/dasolo.online

**Comment by jspenguin2017** [comment]
@uBlock-user You can buy a domain but not have an "a record". Not sure who will do that, but it's possible.

**Comment by uBlock-user** [comment]
> You can buy a domain but not have an "a record"

Yes, if you do that, the website won't be accessible and will stay dead, and the website was running when I posted the filters, on top of that check the link I posted above which fetches the previous DNS records before the website went down, which proves DNS records do exist for that website.

**Comment by jspenguin2017** [comment]
@uBlock-user 
Anyway, my browser tells me it couldn't get DNS record for that domain.

**Comment by uBlock-user** [comment]
Works fine here, I use who.is for fetching DNS records.

**Comment by 4rres** [comment]
If you use the DNS found by uBlock-user the site will work fine, 0 error ;)
